README.md

```diff
@@ -150,11 +150,11 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/><br/>
 
-## ![✔] 2.6 Shut the process gracefully when a stranger comes to town
+## ![✔] 2.6 Shutdown the process gracefully when a stranger comes to town
```

Comment:
Not sure, maybe `Shut down` instead of `Shutdown`?

Reply:
Hmm how about swapping it for 'exit'?

Reply:
+1 for shut down 

Using bold and interesting words contributes to text attractiveness, It's not only about using the right grammar and vocabulary